inputs.go

```diff
@@ -31,6 +48,15 @@ func (in *Inputs) Digest() (*digest.Digest, error) {
 		digests[i] = fdigest
 	}
 
+	if in.inputString.Exists() {
```

Comment:
```suggestion
	if in.inputString != nil && in.inputString.Exists() {
```
The constructor does not initialize inputString. If `Digest()` is called afterwards without calling `SetInputString()` a segfault would happen.

Reply:
This has been resolved in b756e3b